Gaijinizer patch/Map061.txt

```diff
@@ -198,96 +217,103 @@ E-003を投下する前にあの男が死ねば…
 少なくとも、E-003を解き放つ必要はなくなる。
 そういうこったろ？
 > CONTEXT: Map061/events/5/pages/0/123/Dialogue/other2
-If the man dies before dropping E-003 ...
-At least, there is no need to release E-003.
-Is that what happened?#MTLed
+At the very least, if that man dies before E-003
+gets airdropped in, there'll be no need to release
+it, right?#Ron
 > END STRING
 
 > BEGIN STRING
 お前………。
 > CONTEXT: Map061/events/5/pages/0/128/Dialogue/other2
-you……….#MTLed
+You can't...#Ron
 > END STRING
 
 > BEGIN STRING
 分かってるよ。
 
 命令違反者が、最悪殺されるってことくらい…
 > CONTEXT: Map061/events/5/pages/0/130/Dialogue/other2
-I know. The order violator is about to be killed
-in the worst ...#MTLed
+I know.
+
+Worst case, anyone who goes against orders gets
+killed...#Ron
 > END STRING
 
 > BEGIN STRING
 だけど、俺はそれ以上に…
 
 自分の手で、死神みてぇなE-003を解き放ちたくねぇ…
 > CONTEXT: Map061/events/5/pages/0/134/Dialogue/other2
-But I am more than that...With my own hands, I
-want to unleash the evil god E-003 ...#MTLed
+But at the very least I...
+
+I don't want these hands of mine to be the ones to
+release that angel of death, E-003...#Ron
 > END STRING
 
 > BEGIN STRING
 金のためならどんな悪事だってやってきた……
 
 だけど………だけどよ……
 > CONTEXT: Map061/events/5/pages/0/138/Dialogue/other2
-What a bad thing for money ...
-But...... but......#MTLed
+To release such a calamity just for money...
+
+But... But I...#Ron
 > END STRING
 
 > BEGIN STRING
 わかった、もういい。
 
 ………お前の好きにしろ。
 > CONTEXT: Map061/events/5/pages/0/142/Dialogue/other2
-I understand, it's good....... love you.#MTLed
+Okay okay! I get it.
+
+... Just do what you want, then.#Ron
 > END STRING
 
 > BEGIN STRING
 『人を１人殺してでも、阻止したい。』
 
 その覚悟……俺も一緒に責任をとってやる。
 > CONTEXT: Map061/events/5/pages/0/147/Dialogue/other2
-"I want to stop even if I kill one person. "
-Its preparedness...I will take responsibility
-as well.#MTLed
+I'm prepared to kill someone if it'll stop this.
+
+I'll take full responsibility as well.#Ron
 > END STRING
 
 > BEGIN STRING
 相棒……
 > CONTEXT: Map061/events/5/pages/0/151/Dialogue/other2
-Partner……#MTLed
+...#Ron
 > END STRING
```

Comment:
What's your reasoning for leaving this out?

Reply:
This was about a month ago now so I don't entirely remember. But I think it was because it was a Japan-ism that sounds pretty unnatural when you translate it into English. My thinking was that in an English conversation, the other person would probably just be silent rather than say something like "Partner".